src/chapters/configuring.xml

```diff
@@ -3009,13 +3009,102 @@ ed272546-87bd-4db9-acba-e36e1a9ca20a
                                             <primary>SSH</primary>
                                             <secondary>color scheme</secondary>
                                         </indexterm>The color scheme to use for the terminal
-                                        emulator used by SSH connections. Each color scheme dictates
-                                        the default foreground and background color for the
-                                        terminal. Programs which specify colors when printing text
-                                        will override these defaults.</para>
-                                    <para>This parameter is optional. By default, Guacamole will
-                                        render text as gray over a black background.</para>
-                                    <para>Possible values are:</para>
+                                        emulator used by SSH connections. It consists of a
+                                        semicolon-separated series of name-value pairs, separated by
+                                        a colon, that assign default values to colors in the
+                                        terminal emulator palette. Note that programs that specify
+                                        colors when printing text can still override these defaults.
```

Comment:
How does this work in practice?  I understand that if you specify `foreground` and `background` that those could be overridden by the program (e.g. `ls -l --colors=auto`, which would list a directory with some color-coding outside of the specified foreground color), but, if, as in your example below, you change `color9` to a purple shade, presumably any application that goes to use `color9` to print red would, instead, print in purple?

Reply:
Yeah that was language left over from the current manual. Re-reading it, I don't think it adds much value to the section, so I decided to delete that sentence altogether.